storybook/src/stories/Expandable.stories.svelte

```diff
@@ -26,9 +26,5 @@ setTemplate(template);
     {#snippet title()}
       Title
     {/snippet}
-    <!-- eslint-disable-next-line sonarjs/no-unused-vars -->
```

Comment:
After re reading the documentation, I think only `{#snippet children()}` and `{/snippet}` should be removed.

Asking @benoitf for confirmation

Reply:
Agreed, the line with `Children` should be kept. The rule is saying that the `#snippet` wrapper is unnecessary since any content not in a snippet will be set to the `children` snippet anyway.